src/signalfx_metadata.py

```diff
@@ -1054,6 +1053,8 @@ def get_aws_info(host_info={}):
         }
         for k, v in iter(want.items()):
             host_info["aws_" + k] = identity[v]
+        AWS = True
+        set_aws_url(host_info)
```

Comment:
log a message here that you are an aws box for debugging purposes similar to the one below.  It would be good to be able to see if we're flip flopping.